client/src/components/Chat/Input/Input.module.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  margin: 40px;
+}
+.input {
+  background-color: #ffffff;
+  border-radius: 5px;
+  width: 100%;
+  height: 100%;
+}
+.button {
+  height: 42px;
+}
+.inputContainer {
+  display: flex;
+  align-items: stretch;
+}
+@media (max-width: 576px) {
```

Comment:
same as above, all screen values for media queries should be taken from constants